store/tikv/kv.go

```diff
@@ -150,6 +150,18 @@ func NewKVStore(uuid string, pdClient pd.Client, spkv SafePointKV, client Client
 	return store, nil
 }
 
+// SetRegionCacheStore is used to set a store in region cache, for testing only
+func (s *KVStore) SetRegionCacheStore(id uint64, storeType tikvrpc.EndpointType, state uint64, labels []*metapb.StoreLabel) {
```

Comment:
you can add testing only methods to StoreProbe ( https://github.com/pingcap/tidb/blob/4d7fcbb3dffb7372150e47257842eb30efd33bf7/store/tikv/test_probe.go#L32 ) to avoid polluting `KVStore`.